### DISCUSSION TITLE
SFT with qlora-pipe?

### DISCUSSION THREAD

**Discussion by evilyesh** [tdrussell/qlora-pipe#60] [Q&A]
I want to express my deep gratitude for the great work done, it's awesome!
I was able to run the 14B model for finetune!

I apologize for the possibly stupid question, please tell me how to perform Supervised Fine-tuning using your framework?
In the dataloader I saw only DPO and Seq2Seq, but for SFT I need to specially mask the labels.
Can you help, point me in the right direction?
